Lib/pdb.py

```diff
@@ -1593,19 +1593,51 @@ def _runscript(self, filename):
 # Simplified interface
 
 def run(statement, globals=None, locals=None):
+    """Execute the statement (given as a string or a code object) under
+    debugger control.
+
+    The debugger prompt appears before any code is executed; you can set
+    breakpoints and type continue, or you can step through the statement
+    using step or next.
+
+    The optional globals and locals arguments specify the environment in
+    which the code is executed; by default the dictionary of the module
+    __main__ is used (see the explanation of the built-in exec() or
+    eval() functions.).
+    """
     Pdb().run(statement, globals, locals)
 
 def runeval(expression, globals=None, locals=None):
+    """Evaluate the expression (given as a string or a code object)
+    under debugger control.
+
+    When runeval() returns, it returns the value of the expression.
+    Otherwise this function is similar to run().
+    """
     return Pdb().runeval(expression, globals, locals)
 
 def runctx(statement, globals, locals):
     # B/W compatibility
     run(statement, globals, locals)
 
 def runcall(*args, **kwds):
```

Comment:
Hmm, it's not clear to me why the parameter *function* is missing from the function, as it's present in the documentation: https://docs.python.org/3/library/pdb.html#pdb.runcall. Unfortunately, we don't have a presently active [expert](https://devguide.python.org/experts/#miscellaneous) for the pdb module to clarify this point.

This is unrelated to the PR so I don't think it should affect whether or not it's merged, but the discrepancy here is worth noting.